### DISCUSSION TITLE
Filter functionality is case sensitive when searching on Postgres because LIKE is used instead of ILIKE

### DISCUSSION THREAD

**Discussion by bamboowonder** [directus/directus#12662] [Feature Requests]
### Describe the Bug

The "contains" filter must be using a WHERE LIKE '%somesearchterm%' which is a case sensitive search.
Ideally filtering would be case insensitive.
This can be achieved by using ILIKE instead of LIKE in Postgres.
------
This is somewhat related to #12658 that alternatively a citext column can be used in postgres, however those aren't recognized correctly by Directus.

### To Reproduce

Attempt to use a "contains" filter on a column that includes capital letters.   
Note that it is using a case insensitive search.

<details>
<summary>More info</summary>

### What version of Directus are you using?

9.8.0

### What version of Node.js are you using?

16.14.2

### What database are you using?

postgres 14

### What browser are you using?

chrome

### How are you deploying Directus?

CLI npm package

</details>

**Comment by azrikahar** [comment]
This has been discussed in #2981 for a while now, but the main issue has been to make it work for every supported database vendors rather than only making it work for Postgres, and that's where knex/knex#2265 comes in (since Directus is using knex under the hood).

The good news is that issue was resolved fairly recently and there is now the open PR #11737 which leverages it!

**Reply by bamboowonder** [reply]
The thing is, postgres has citext column types, which look like they'll be recognized properly in the next release. #12670 
With citext, it is case-insensitive so it doesn't matter what directus/knex is sending in the query.
And yet, when I do a search in the interface against a postgres citext column, it is case sensitive.
so that is odd.  if i run a query like that directly on the database, it behaves as expected.
Is there some sort of caching I need to purge?  in my .env I have caching of everything disabled.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
